README.md

```diff
@@ -773,11 +829,11 @@ Feature: Bob can only read an RDF resource to which he is only granted read acce
         const testContainer = createTestContainer();
         const resource = testContainer.createChildResource('.ttl', karate.readAsString('../fixtures/example.ttl'), 'text/turtle');
         if (resource.exists()) {
-          const acl = aclPrefix
-            + createOwnerAuthorization(webIds.alice, resource.getUrl())
-            + createBobAccessToAuthorization(webIds.bob, resource.getUrl(), 'acl:Read')
-          karate.log('ACL: ' + acl);
-          resource.setAccessDataset(acl);
+          const access = resource.getAccessDatasetBuilder(webIds.alice)
+            .setAgentAccess(resource.getUrl(), webIds.bob, ['read'])
+            .build();
+          karate.log('ACL:\n' + access.asTurtle());
+          resource.setAccessDataset(access);
```

Comment:
This example too seems very verbose for what it is doing. It exposes the test writer to things that I feel should be internals, like that you need a `readAsString` call, and the test that the resource exists should always be performed, so that should be something the test writer shouldn't be exposed to, IMHO.